control/veda_val.py

```diff
@@ -202,14 +181,7 @@ def send_val_data(self):
             'courses': val_courses
         }
 
-        r1 = requests.get(
-            '/'.join((
-                self.auth_dict['val_api_url'],
-                self.video_proto.val_id
-            )),
-            headers=self.headers,
-            timeout=self.auth_dict['global_timeout']
-        )
+        r1 = self.oauth2_client.request('GET', '/'.join((self.auth_dict['val_api_url'], self.video_proto.val_id, '')))
```

Comment:
The trailing `/` didn't exist before - is this OK?

Reply:
Thanks for catching this...I don't think it hurts but it doesn't need to be there.  I will remove.